README.md

```diff
@@ -1,78 +1,91 @@
-
-## Testing
-
-There is no boilerplate for testing in this starter-kit repo. You will need to set this up yourself. However, if you ran `npm install`, then the tooling you need to start testing is already installed (`mocha` and `chai`).
-
-
-## Data Model
-
-**Users**
-
-```
-[
-  {
-    "id": [number],
-    "name": [string],
-    "address": [string],
-    "email": [string],
-    "strideLength": [number - feet],
-    "dailyStepGoal": [number - steps],
-    "friends": [array - one-way connection to other user(s)]
-  },
-  ...more user data
-]
-```
-
-**Activity**
-
-```
-[
-  {
-    "userID": [number],
-    "date": [string YYYY/MM/DD],
-    "numSteps": [number - steps],
-    "minutesActive": [number - minutes],
-    "flightsOfStairs": [number - flights]
-  },
-  ...more activity data
-]
-```
-
-**Hydration**
-
-```
-[
-  {
-    "userID": [number],
-    "date": [string YYYY/MM/DD],
-    "numOunces": [number - ounces]
-  },
-  ...more hydration data
-]
-```
-
-**Sleep**
-
-```
-[
-  {
-    "userID": [number],
-    "date": [string YYYY/MM/DD],
-    "hoursSlept": [number - hours],
-    "sleepQuality": [number - unitless]
-  },
-  ...more sleep data
-]
-```
+<img width="230" alt="Health Hub Lable" src="https://user-images.githubusercontent.com/95496577/169706412-03739a11-373c-439f-998f-174dde44173f.png"> 
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Abstract](#abstract)
+- [Illustration](#illustration)
+- [Wireframe](#wireframe)
+- [Technologies Used](#technologies-used)
+- [Context and Features](#context-and-features)
+- [Lessons Learned](#lessons-learned)
+- [Future Features](#future-features)
+- [Application Set-Up](#application-set-up)
+- [Contributor LinkedIn and GitHub](#contributor-linkedin-and-github)
+- [Project Specs](#project-specs)
+
+## Abstract
+
+- The _HealthHub_ Project was designed to test my fellow project collaborators and I on our abilities to implement ES6 classes, use array and object prototype methods for data manipulation, create an easy to use and understand dashboard with clearly displayed information, make network requests to retrieve data and practice our TDD skills with Mocha/Chai. It’s a health and wellness application that showcases wellness information for a random user. 
+
+## Illustration
+
+![HealthHub Recording](https://user-images.githubusercontent.com/95496577/169706726-6dc58550-073f-4af8-89a8-c24313bcb7bc.gif)
+
+## Wireframe
+
+- [Figma Wireframe](https://www.figma.com/file/MAcq55QYBdenQxU3mBVcVz/FitLit-Wireframe?node-id=0%3A1) 
+
+<img width="721" alt="New Wire Frame Layout" src="https://user-images.githubusercontent.com/95496577/169707014-ea0f4135-dca2-4ea0-8bbe-2e4e7047f150.png">
+
+## Technologies Used
+
```

Comment:
Maybe we should add Chart.js and Normalize.css to our technologies? Just a suggestion.